data/json/items/comestibles/drink_other.json

```diff
@@ -192,18 +258,24 @@
     "charges": 16,
     "phase": "liquid",
     "fun": -25,
-    "flags": [ "NUTRIENT_OVERRIDE" ],
```

Comment:
Have you checked the nutrition value of crafted vegetable oil with this change? Last time I checked it caused vegetable oil to have no nutrition value at all.

Reply:
Yeah, I added calories and  health data to the appropriate seed types. I did the math in reverse starting from a serving of cooking oil (all types I researched from the recipes have FDA values of 120 calories so I added calorie values to each seed type to give 120 calorie oil. Some seed types are bitter and unenjoyable, like grape seeds, but aren't toxic. Cotton seeds are toxic so eating them raw is quite unhealthy (not immediately deadly, but it's a toxic buildup) so I added a negative health value to eating them. Canola seeds also have a toxic property.